### PR TITLE
Add rune armor items

### PR DESCRIPTION
**Pull request by RedSparr0w**
Add rune armor items
Add rune god armor items

### CODE REVIEW

**Review by SchauweM — DISMISSED** [review]
Thanks! I've approved the PR, even though the quest requirements for the platebodies is missing because we don't have the actual quest in the server yet.

**Comment by Promises** [comment]
> Thanks! I've approved the PR, even though the quest requirements for the platebodies is missing because we don't have the actual quest in the server yet.

if you make a note of that, i can probably set it up to check if a quest exists, if not it will allow

**Comment by RedSparr0w** [comment]
Completely forgot the platebody had a quest requirement 😅

**Comment by Promises** [comment]
I might be wrong, but rune gloves might also be locked behind some special cases related to RFD

**Comment by SchauweM** [comment]
> > Thanks! I've approved the PR, even though the quest requirements for the platebodies is missing because we don't have the actual quest in the server yet.
> 
> if you make a note of that, i can probably set it up to check if a quest exists, if not it will allow

That might actually be a better solution. Do you want me to close this PR or do you want to add the functionality to it in this PR?

**Comment by RedSparr0w** [comment]
I believe the rune gloves don't have any hard requirements other than the fact you can't obtain them before completing the quest.
Although they should be non-tradable.

**Comment by Promises** [comment]
> > > Thanks! I've approved the PR, even though the quest requirements for the platebodies is missing because we don't have the actual quest in the server yet.
> > 
> > 
> > if you make a note of that, i can probably set it up to check if a quest exists, if not it will allow
> 
> That might actually be a better solution. Do you want me to close this PR or do you want to add the functionality to it in this PR?

close this one, but make a note or send me a message on discord in a few hours, and ill work on it separately

File: data/config/items/equipment/standard-metals/rune-armour.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+                    }
+                }
+            }
+        }
+    },
+
+    "rs:rune_gloves": {
+        "game_id": 7460,
+        "examine": "A pair of very nice gloves.",
+        "weight": 0.226,
```
Comment (RedSparr0w) [inline data/config/items/equipment/standard-metals/rune-armour.json:19]:
```suggestion
        "weight": 0.226,
        "tradable": false,
```

**Review by SchauweM — APPROVED** [review]

**Pull request merged**